_data/simple-icons.json

```
@@ -12476,4 +12481,4 @@
             "source": "https://www.zyte.com/"
         }
     ]
-}
+}
```

You need to add a newline character here:

```suggestion
}

```